app/src/routes/shipmentsMap/+layout.svelte

```diff
@@ -122,33 +120,20 @@
 	});
 </script>
 
-<main class="relative h-screen w-full overflow-hidden">
-	<div class="absolute z-10 w-3/4 md:w-1/3 xl:w-1/4 left-1/2 transform -translate-x-1/2 top-4">
-		<div class="m-3 p-0.5 rounded-full bg-gradient-to-r from-primary to-secondary">
-			<label for="name" class="sr-only">Name</label>
-			<input
-				class="px-3 py-1.5 w-full rounded-full bg-background focus:outline-none text-sm lg:text-md"
-				type="text"
-				id="name"
-				placeholder="Search"
-				bind:value={$storeToSearchIn.searchString}
-				on:keyup={handleSearchKeyUp}
-			/>
-		</div>
-	</div>
-	<div class="hidden md:block absolute top-7 right-7 z-40">
-		<WalletMultiButton onClose={() => {}} />
-	</div>
-
-	<div class="hidden md:block">
-		<MapWrapper>
-			<slot />
-		</MapWrapper>
+<div
+	class="hidden md:block absolute z-10 w-3/4 md:w-1/3 xl:w-1/4 left-1/2 transform -translate-x-1/2 top-4"
+>
+	<div class="m-3 p-0.5 rounded-full bg-gradient-to-r from-primary to-secondary">
+		<label for="name" class="sr-only">Name</label>
+		<input
+			class="px-3 py-1.5 w-full rounded-full bg-background focus:outline-none text-sm md:text-lg"
+			type="text"
+			id="name"
+			placeholder="Search"
```

Comment:
i can't find it on mobile, is it there somewhere?

Reply:
i mean no searchbar on mobile

Reply:
I know that. I'm still thinking how to do it.

Reply:
Becouse it should be in Navbar, but this complicates the search you've already created.

Reply:
How so? Keyboard events?

Reply:
I would leave it for later, just open issue to not forget it.